test/parallel/test-sqlite-backup.mjs

```diff
@@ -141,6 +142,46 @@ test('database backup', async (t) => {
   });
 });
 
+test('backup database using location as URL', async (t) => {
+  const database = makeSourceDb();
+  const destDb = pathToFileURL(nextDb());
+
+  await backup(database, destDb);
+
+  const backupDb = new DatabaseSync(destDb);
+  const rows = backupDb.prepare('SELECT * FROM data').all();
+
+  t.assert.deepStrictEqual(rows, [
+    { __proto__: null, key: 1, value: 'value-1' },
+    { __proto__: null, key: 2, value: 'value-2' },
+  ]);
+
+  t.after(() => {
```

Comment:
Can you move this up in the test as far as possible. Otherwise, it's possible that an exception is thrown and this never executes.